DQM/SiPixelPhase1Summary/src/SiPixelPhase1Summary.cc

```diff
@@ -144,6 +144,100 @@ void SiPixelPhase1Summary::bookSummaries(DQMStore::IBooker& iBooker) {
     deadROCSummary->setBinLabel(i + 1, xAxisLabelsReduced_[i]);
   }
 
+  //New residual plots for the PXBarrel separated by inner and outer modules per layer
+  iBooker.setCurrentFolder("PixelPhase1/Tracks/PXBarrel");
+  for (std::string layer : {"1", "2", "3", "4"}) {
+    residuals_["residual_mean_x_Inner_PXLayer_" + layer] =
+        iBooker.book1D("residual_mean_x_Inner_PXLayer_" + layer,
+                       "Mean Track Residuals X Inner Modules for Layer " + layer + ";mean(x_rec-x_pred)[cm]",
+                       100,
+                       -0.1,
+                       0.1);
+    residuals_["residual_mean_x_Outer_PXLayer_" + layer] =
+        iBooker.book1D("residual_mean_x_Outer_PXLayer_" + layer,
+                       "Mean Track Residuals X Outer Modules for Layer " + layer + ";mean(x_rec-x_pred)[cm]",
+                       100,
+                       -0.1,
+                       0.1);
+    residuals_["residual_mean_y_Inner_PXLayer_" + layer] =
+        iBooker.book1D("residual_mean_y_Inner_PXLayer_" + layer,
+                       "Mean Track Residuals Y Inner Modules for Layer " + layer + ";mean(y_rec-y_pred)[cm]",
+                       100,
+                       -0.1,
+                       0.1);
+    residuals_["residual_mean_y_Outer_PXLayer_" + layer] =
+        iBooker.book1D("residual_mean_y_Outer_PXLayer_" + layer,
+                       "Mean Track Residuals Y Outer Modules for Layer " + layer + ";mean(y_rec-y_pred)[cm]",
+                       100,
+                       -0.1,
+                       0.1);
+
+    residuals_["residual_rms_x_Inner_PXLayer_" + layer] =
+        iBooker.book1D("residual_rms_x_Inner_PXLayer_" + layer,
+                       "RMS of Track Residuals X Inner Modules for Layer " + layer + ";rms(x_rec-x_pred)[cm]",
+                       100,
+                       -0.1,
```

Comment:
@dmeuser, why the RMS are binned in the negatives? It's a defined positive quantity, see plots at: https://tinyurl.com/qsb3yst

Reply:
@mmusich, you are right, this makes no sense. I will change it and make a new commit?

Reply:
thx!